tests/testMaskedImage.py

```diff
@@ -120,9 +120,10 @@ def tearDown(self):
 
     def testArrays(self):
         image, mask, variance = self.mimage.getArrays()
-        self.assert_((self.mimage.getImage().getArray() == image).all())
-        self.assert_((self.mimage.getMask().getArray() == mask).all())
-        self.assert_((self.mimage.getVariance().getArray() == variance).all())
+        self.assertListEqual(self.mimage.getImage().getArray().flatten().tolist(), image.flatten().tolist())
```

Comment:
Hmm. You are right. It's not obvious from the `unittest` documentation but the code does try to compare standard types using specialist methods so `assertEqual` would work here.